src/collision-checkers/minkowski.jl

```diff
@@ -381,23 +381,23 @@ function _bounding_radius(veh::Entity{S,D,I}) where {S,D<:AbstractAgentDefinitio
 end
 
 """
-    is_potentially_colliding(A::Entity{VehicleState,D,I}, B::Entity{VehicleState,D,I}) where {D<:AbstractAgentDefinition,I} 
+    is_potentially_colliding(A::Entity{S,D,I}, B::Entity{S,D,I}) where {S,D<:AbstractAgentDefinition,I} 
```

Comment:
Can we assume that every state type `S` has a field called `state`?

Reply:
Every `Entity` has a field called `state`

Reply:
Ouh true, my bad, perfect